vars/stageDeployToOpenshift.groovy

```diff
@@ -12,6 +12,28 @@ def call(def context) {
       destTag: "latest",
       namespace: context.targetProject
     )
+
+    def ocpLatestImage = sh(returnStdout: true, script:"oc get istag ${context.componentId}:latest -n ${context.targetProject} -o jsonpath='{.image.dockerImageReference}'", label : "find last image").trim()
+      
+    def ocpDeployment
+    timeout(context.openshiftBuildTimeout) {
```

Comment:
I think this is "misusing" the build timeout setting. Right now this is meant to stop the container build if it takes too long. Therefore it needs to be configurable because ODS cannot know how long a container image build will take.

Here, the allowed time will be different because the time it takes to boot the service is not equal to the time it takes to build it. 

Therefore I suggest to have a separate timeout parameter here ...